app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -80,6 +81,5 @@ class AuthRepository @Inject constructor(
 		return userProfile()
 	}
 
-	private fun requireUser() =
-		firebaseAuth.currentUser!!
+	private fun requireUser(): FirebaseUser = checkNotNull(firebaseAuth.currentUser) { "The Firebase user is null" }
```

Comment:
I don't see a point in changing `!!` to `checkNotNull`. The app will crash in both cases when the user is null but the solution with `!!` is shorter and easier to understand what's is going on.